packages/editor/src/browser/editor.ts

```diff
@@ -110,7 +110,7 @@ export interface SetDecorationParams {
 }
 
 export interface DeltaDecorationParams {
-    uri: string;
+    uri?: string;
```

Comment:
How about removing it?

Reply:
@akosyakov it depends on, how to deal with diff editor. should we just let the implementor do the work to get to the wanted editor part of the diff editor? then `deltaDecorations` of the diff editor would be no-op? Or should it log a warning, or even throw?

Reply:
I removed it, and changed diff editor's `deltaDecorations` to be a no-op.